fe/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -452,11 +452,24 @@ public void getLoadJobInfo(Load.JobInfo info) throws DdlException {
         }
     }
 
-    public void submitJobs() {
+    public void prepareJobs() {
+        submitJobs();
+        analyzeLoadJobs();
```

Comment:
It is strange that you submit job first and analyze job later.

Reply:
Changed